modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -6780,6 +6817,57 @@ private void _testGetNestedFieldDetailsInRelationships(
 			expectedFieldName, objectFieldNamesAndObjectFieldValues, type);
 	}
 
+	private void _testPatchCustomObjectEntryExternalReferenceCode(
+			String endpoint, String externalReferenceCode)
+		throws Exception {
+
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"externalReferenceCode", externalReferenceCode
+			).toString(),
+			endpoint, Http.Method.PATCH);
+
+		Assert.assertEquals(
+			jsonObject.getString("externalReferenceCode"),
+			externalReferenceCode);
+	}
+
+	private void _testPatchCustomObjectEntryWithDuplicateExternalReferenceCode(
+			String objectDefinitionRESTContextPath)
+		throws Exception {
+
+		Assert.assertEquals(
+			200,
+			HTTPTestUtil.invokeToHttpCode(
+				JSONUtil.put(
+					_OBJECT_FIELD_NAME_1, RandomTestUtil.randomString()
+				).put(
+					"externalReferenceCode", _ERC_VALUE_1
+				).toString(),
+				objectDefinitionRESTContextPath, Http.Method.POST));
+
+		Assert.assertEquals(
+			200,
+			HTTPTestUtil.invokeToHttpCode(
+				JSONUtil.put(
+					_OBJECT_FIELD_NAME_1, RandomTestUtil.randomString()
+				).put(
+					"externalReferenceCode", _ERC_VALUE_2
+				).toString(),
+				objectDefinitionRESTContextPath, Http.Method.POST));
+
+		Assert.assertEquals(
+			400,
+			HTTPTestUtil.invokeToHttpCode(
```

Comment:
Furthermore, we would still have the LPP issue. Please, read the LPP and talk to PM to decide which is the expected behaviour when a user send a PUT request with different ERC. It is not clear to me

Reply:
hey @ccorreagg! I've been talking with PM and after making some tests in Object Entries PUT operations, we found that it updates the ERC using the byId and byERC endpoints. For that reason, we should able to PATCH Object Entries ERC byId and byERC endpoints. You can check it in the LPD description, PM has updated it